ngraph/frontend/onnx/frontend/src/core/transform.cpp

```diff
@@ -38,19 +50,29 @@ ONNX_NAMESPACE::TypeProto get_input_type(std::string const& name, ONNX_NAMESPACE
 void ngraph::onnx_import::transform::expand_onnx_functions(ONNX_NAMESPACE::ModelProto& model_proto) {
     auto graph_proto = model_proto.mutable_graph();
 
-    for (int i = 0; i < graph_proto->node().size(); ++i) {
-        ONNX_NAMESPACE::NodeProto node = graph_proto->node().Get(i);
+    ONNX_NAMESPACE::GraphProto new_graph;
+    int origin_graph_index = 0;
+    int new_graph_index = 0;
+    while (origin_graph_index < graph_proto->node().size() || new_graph_index < new_graph.node().size()) {
```

Comment:
You can consider using https://en.cppreference.com/w/cpp/algorithm/rotate to move all new added nodes to the correct position.